runtime-modules/project-token/src/lib.rs

```diff
@@ -98,23 +99,25 @@ decl_storage! {
             hasher(blake2_128_concat) T::Hash => ();
 
         /// Bloat Bond value used during account creation
-        pub BloatBond get(fn bloat_bond) config(): ReserveBalanceOf<T>;
+        pub BloatBond get(fn bloat_bond) config(): JoyBalanceOf<T>;
     }
 
     add_extra_genesis {
         build(|_| {
-            // We deposit some initial balance to the pallet's module account on the genesis block
-            // to protect the account from being deleted ("dusted") on early stages of pallet's work
+            // We deposit some initial balance to the pallet's module account
+            // (and sub-accounts!!) on the genesis block to protect the account
```

Comment:
Expanding on the previous comment:
The only reason why this works in the test environment is because of the `u64` mock for `AccountId`.
Normally, you'd need to initialize a sub-account for each `token_id` separately, which is just not feasible, so I suggest using a single module account for all purposes.